### DISCUSSION TITLE
Loading system fonts

### DISCUSSION THREAD

**Discussion by ghost** [protectwise/troika#185] [Q&A] [answered]
Is there a way to load a system font like "Arial"?
It seems that I can't simply send in the font name and it load (rather than a font URL).

**Comment by lojjic** [comment, marked as answer]
Nope, unfortunately there is no browser API that lets us get access to the data of system fonts. There was a [work in progress spec](https://wicg.github.io/local-font-access/) for allowing that but it doesn't seem to have gained much traction and it requires the user to explicitly grant permission anyway to avoid privacy/fingerprinting concerns.

**Reply by ghost** [reply]
Wow what a fast response! Thanks!
